example/js/main.mjs

```diff
@@ -1,7 +1,13 @@
-import { ICU4XFixedDecimal } from "./high-level.mjs";
+import { ICU4XFixedDecimal, ICU4XLocale } from "./high-level.mjs";
```

Comment:
thought: ooh, cross-module imports are going to be interesting once we have pluralrules/etc

probably will be another list you accumulate to same as the `pre_stmts` stuff you're doing